app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,53 @@
+<!--
+  ~ Copyright 2018 Google LLC
```

Comment:
Can you update the copyright year to 2019?

Reply:
Done